plugin/src/withIos.ts

```diff
@@ -0,0 +1,24 @@
+import { withPlugins, withPodfile } from '@expo/config-plugins'
+import type { ExpoConfig } from '@expo/config-types'
+import type { Props } from '.'
+
+const withIosPodfile = (props: Props) => (expoConfig: ExpoConfig) =>
+  withPodfile(expoConfig, (c) => {
+    if (!props.ios?.buildStatic) return c
+    if (c.modResults.contents.includes('Pod::BuildType.static_library')) return c
```

Comment:
this is dangerous. If you now change the list it won't update.

What if we assign a variable in the podspec that contains the list. Then we check if there is a variable assignment (.includes('_animoIdMdocDataTransferStaticLibraries =')). If it doesn't, we add the full new new code, if it does we just replace the line that sets the array list. That way we can update the code.

Also -- does it work to  just add it to the end using `+=`?